Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3
+
+ENV PYTHONDONTWRITEBYTECODE 1
```

Comment:
this will make black's startup and execution significantly slower

```console
$ time PYTHONDONTWRITEBYTECODE=1 black t.py 
All done! ✨ 🍰 ✨
1 file left unchanged.

real	0m0.233s
user	0m0.221s
sys	0m0.012s

$ time black t.py   # this one will compile and write the bytecode
All done! ✨ 🍰 ✨
1 file left unchanged.

real	0m0.238s
user	0m0.207s
sys	0m0.028s

$ time black t.py 
All done! ✨ 🍰 ✨
1 file left unchanged.

real	0m0.130s
user	0m0.123s
sys	0m0.004s
```
```